services/static-webserver/client/source/resource/osparc/blacklist.json

```diff
@@ -0,0 +1,181 @@
+{
+  "lite": [
```

Comment:
why not apply these to *all* lite the products?

Reply:
It applies to all ``lite`` products that have a request account form. ``s4llite``, for now goes through the form in the ZMT website.